src/components/main/TeamMember.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Image from "next/image";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Link from "next/link";
+
+interface SocialLink {
```

Comment:
You can move this up into a shared location and use it in the Team page. I made a comment there about creating a type for a property.